packages/members-api/lib/stripe/index.js

```diff
@@ -177,6 +178,40 @@ module.exports = class StripePaymentProcessor {
         });
     }
 
+    async setComplimentarySubscription(member) {
+        const subscriptions = await this.getActiveSubscriptions(member);
+        const complimentaryPlan = this._plans.find(plan => (plan.nickname === 'Complimentary'));
+
+        const customer = await this._customerForMemberCheckoutSession(member);
+
+        if (!subscriptions.length) {
+            const subscription = await create(this._stripe, 'subscriptions', {
+                customer: customer.id,
+                items: [{
+                    plan: complimentaryPlan.id
+                }]
+            });
+
+            await this._updateSubscription(subscription);
+        } else {
+            // NOTE: we should only ever have 1 active subscription, but just in case there is more update is done on all of them
+            for (const subscription of subscriptions) {
+                const updatedSubscription = await update(this._stripe, 'subscriptions', subscription.id, {
+                    proration_behavior: 'none',
+                    plan: complimentaryPlan.id
```

Comment:
Does this definitely remove the previous plan for the subscription? I remember having to do something like this to get plans updating correctly:

```js
stripe.subscriptions.update(subscription.id, {
  items: [{
    id: subscription.items.data[0].id,
    plan: 'blahblah'
  }]
})
```

But it may have been a different usecase

Reply:
Just double checked this. In both the Stripe API response and on Stripe dashboard UI the plan is set to the 'Complimentary' one.
![Screenshot from 2020-01-27 12-24-56](https://user-images.githubusercontent.com/675397/73151929-21bc1080-4100-11ea-8286-83f34c4aa363.png)

IIRC, I didn't go with the `items:[{}]` approach because it needed "subscription items" which I didn't feel like necessary to re-fetch on every update.